Make text-shadow-orientation-upright-001-ref.html avoid sub-pixel anti-alias issue

TalbotG opened the pull request:
This is a followup-to [a comment in PR27770](https://github.com/web-platform-tests/wpt/pull/27770#issuecomment-954686285) which went like this:

> I'm a little bit concerned that the text will have anti-aliasing but the image reference won't, so probably a better reference would be to have a horizontal-mode copy of the text-shadow test code.
https://wpt.fyi/results/css/css-writing-modes/text-shadow-orientation-upright-001.html?label=experimental&label=master&aligned
There is indeed a failure in Edge 96 due to anti-aliasing but only in Edge 96.

The proposed modification to 

/css/css-writing-mode/reference/text-shadow-orientation-upright-001-ref.html

creates an horizontal-mode only copy of the text-shadow test code.

Over at my website:

http://www.gtalbot.org/BrowserBugsSection/CSS3WritingModes/s76-text-shadow-orientation-upright-002.html

New reference:
http://www.gtalbot.org/BrowserBugsSection/CSS3WritingModes/reference/text-shadow-orientation-upright-002-ref-new.html

TalbotG committed:
Tried to make text-shadow-orientation-upright-001-ref.html avoid sub-pixel anti-alias issue

TalbotG committed:
Forgot to add href='/fonts/ahem.css'

TalbotG commented:
Hm... is there a way to check if the stable or latest Edge version passes this test now? ... and not wait for a week about this... because Edge's test failure was the reason why I created this PR. 
Anyway... minor issue... we'll see in a few days..

upsuper commented:
By the way, I think the reference page on your website doesn't contain the necessary Ahem style.

TalbotG commented:
Xidorn,
As a regular test creator for the last 2 decades, I just downloaded the Ahem font and installed it. Therefore, I never had the need to write the
  <link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
for my tests on my website. And also assumed that test creators and test reviewers already had the Ahem font installed on their computers.

Btw, I even created this page of the Ahem font variants 6 years ago:

http://www.gtalbot.org/BrowserBugsSection/CSS3WritingModes/Ahem2016TestPage.html

mentioned in

https://lists.w3.org/Archives/Public/public-css-testsuite/2016Aug/0005.html

Anyway. Over at my website, I created a /font directory, added ahem.css and Ahem.ttf files and adjusted accordingly 
http://www.gtalbot.org/BrowserBugsSection/CSS3WritingModes/reference/text-shadow-orientation-upright-002-ref-new.html

So things should be normal for you now. I will adjust ruby-abs-pos files too in pull/35361 in 5 min.

TalbotG commented:
> ... because Edge's test failure was the reason why I created this PR.

https://wpt.fyi/results/css/css-writing-modes/text-shadow-orientation-upright-001.html?label=experimental&label=master&aligned&view=subtest

Edge 105 now passes this test!!

upsuper commented:
> As a regular test creator for the last 20+ years, I just downloaded the Ahem font and installed it on my computer. Therefore, I never had the need to write the
> `<link rel="stylesheet" type="text/css" href="/fonts/ahem.css">`
> for my own tests on my website.

The web today is certainly very different from 20 years ago. 20 years ago we didn't have web fonts, and thus having test fonts installed locally is the only possible approach to get it work. Now it's no longer the case.

My understanding of the wpt document regarding [Ahem font](https://web-platform-tests.org/writing-tests/ahem.html) is that tests are expected to include the `<link>` if they use Ahem font, and there is no expectation that the environment running wpt have the font pre-installed.

> And I also assumed that test creators and test reviewers already had the Ahem font installed on their computers, just like me

I would suggest you to remove the font from your computer as well.

The Ahem font has evolved several times in the past, added some useful special mappings, and I don't think there is any guarantee that it wouldn't be updated again in the future. Font installed locally may go outdated when that happens again, and cause confusion.
